languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10026,6 +10026,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_OU_QU_ELLE" name="quelle ou qu'elle" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="SENT_START" postag_regexp="yes"></token>
+                  <marker>
+                      <token regexp="yes">Qu(')?</token>
+                      <token regexp="yes">elle</token>
+                  </marker>
+                  <token postag="V.*" postag_regexp="yes" skip="-1"><exception postag="V.* sub" postag_regexp="yes"></exception></token>
```

Comment:
The postag in the exception contains an error: It should be `"V.* sub .*"`, otherwise it won't catch anything. It would be best if you put an example at the end for the kind of sentence you want to exclude. That way you can see whether the exception works.